@xen-orchestra/web-core/lib/icons/fa-icons.ts

```diff
@@ -210,6 +211,7 @@ export const faIcons = defineIconPack({
   star: { icon: faStar },
   stop: { icon: faStop },
   tags: { icon: faTags },
+  template: { icon: faPuzzlePiece },
```

Comment:
```suggestion
  puzzle-piece: { icon: faPuzzlePiece },
```
@ByScripts  what do you think ?